### PR TITLE
Add AU eRequesting Placer CapabilityStatement

### PR DESCRIPTION
**Pull request by dbojicic**
AU eRequesting Placer CapabilityStatement, as agreeed in https://confluence.hl7.org/pages/viewpage.action?pageId=256181872.

Changed include:

- add AU eRequesting Placer  CapabilityStatement (with generated narrative)
- update CapabilityStatement page
- update index.html
- update eRequesting Placer actor

### CODE REVIEW

File: input/resources/au-erequesting-placer.xml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+<CapabilityStatement xmlns="http://hl7.org/fhir">
+  <id value="au-erequesting-placer"/>
+  <extension url="http://hl7.org/fhir/StructureDefinition/structuredefinition-fmm">
+    <valueInteger value="0"/>
+  </extension>
+  <url value="http://hl7.org.au/fhir/ereq/CapabilityStatement/au-erequesting-placer"/>
+  <name value="AUeRequestingPlacerCapabilityStatement"/>
+  <title value="AU eRequesting Placer CapabilityStatement"/>
+  <status value="draft"/>
+  <description value="This CapabilityStatement describes the basic rules for the [AU eRequesting Placer actor](ActorDefinition-au-erequesting-actor-placer.html) that is responsible for initiating and creating diagnostic service requests. The complete list of FHIR profiles, RESTful operations, and search parameters supported by AU eRequesting clients are defined in this CapabilityStatement."/>
```
Comment (dt-r) [inline input/resources/au-erequesting-placer.xml:11]: replace "AU eRequesting clients" with "AU eRequesting placers"

File: input/resources/au-erequesting-placer.xml [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    </security>
+    <!-- SERVICEREQUEST :: DONE -->
+    <resource>
+      <extension url="http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation">
+        <valueCode value="SHALL"/>
+      </extension>
+      <type value="ServiceRequest"/>
+      <supportedProfile value="http://hl7.org.au/fhir/ereq/StructureDefinition/au-erequesting-servicerequest">
+        <extension url="http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation">
+          <valueCode value="SHALL"/>
```
Comment (dt-r) [inline input/resources/au-erequesting-placer.xml:53]: All of these profiles are SHOULD not SHALL.

File: input/resources/au-erequesting-placer.xml [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+        </extension>
+      </supportedProfile>
+      <supportedProfile value="http://hl7.org.au/fhir/ereq/StructureDefinition/au-erequesting-servicerequest-imag">
+        <extension url="http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation">
+          <valueCode value="SHALL"/>
+        </extension>
+      </supportedProfile>
+      <documentation value="Servers **SHALL** support the ServiceRequest resource, at least one AU eRequesting ServiceRequest profile, and the conformance expectations for the ServiceRequest resource."/>
```
Comment (dt-r) [inline input/resources/au-erequesting-placer.xml:66]: Still says "Servers"

**Review by dt-r — CHANGES_REQUESTED** [review]
See inline comments

File: input/resources/au-erequesting-placer.xml (hunk 3 above)
Comment (dbojicic) [reply input/resources/au-erequesting-placer.xml]: Fixed.

File: input/resources/au-erequesting-placer.xml (hunk 2 above)
Comment (dbojicic) [reply input/resources/au-erequesting-placer.xml]: Fixed.

**Review by dt-r — APPROVED** [review]

**Pull request merged**